lib/csetgrp.gi

```diff
@@ -788,7 +788,34 @@ local a,r;
   TryNextMethod();
 end);
 
-
+InstallMethod(Intersection2, "general groups", IsIdenticalObj,
+              [IsRightCoset,IsRightCoset],
+function(cos1,cos2)
+    local H1, H2, x1, x2, shift, sigma, TheInt, CosTest, eCos, rho;
```

Comment:
local variables usually should start with a lower case latter (we make exceptions for things like `G` or `H1`, but that's usually about it). So `theInt` or `the_int` instead of `TheInt` etc.

Reply:
Ok, I renamed the variables according to the standard.